app/models/post.rb

```diff
@@ -0,0 +1,22 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :likes
+  has_many :comments
+
+  validates :author, presence: true
+  validates :title, presence: true
+  validates :text, presence: true
+  validates :commentscounter, presence: true
+  validates :likescounter, presence: true
+
+  def self.update_post_counter(author)
+    user = User.find_by(name: author)
+    user.postscounter = user.posts.count
+    user.save
+  end
+
+  def self.five_most_recent_comment(title)
+    post = Post.find_by(title:)
+    post.comments.order(created_at: :desc).limit(5)
+  end
```

Comment:
- [ **OPTIONAL** ] What do you think if you just update the above method to some thing like the example below for returning the last 5 comments?
```ruby
  ...

def update_comments_counter
  post.increment!(:comments_counter)
end
```

Reply:
Thanks, but it seems you tagged the wrong lines of code